src/components/navigation-header/components/home-page-content/home-page-content.module.css

```diff
@@ -1,13 +1,16 @@
 .siteLogo {
   display: flex;
 
-  /* Note: padding here necessary to match height of .productLogo
+  /* Note: min-height necessary to match height of .productLogo
      in navigation-header.module.css */
-  padding-top: 4px;
-  padding-bottom: 4px;
+  min-height: 40px;
```

Comment:
Since the height of the `svg` can vary, I'm using `min-height` here to achieve the same effect as before.

Without this adjustment, the height of the top nav bar varies on the home page due to the different logo size. An alternate approach might be to set exact padding values with a media query; however this seemed more opaque and less maintainable.

Reply:
Should the logo be vertically centered?

<img width="373" alt="CleanShot 2022-06-15 at 10 04 48@2x" src="https://user-images.githubusercontent.com/825855/173847110-699f5eb4-2203-47a5-9400-e143c5d55031.png">